ddl/table.go

```diff
@@ -379,6 +379,28 @@ func onModifyTableComment(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 	return ver, nil
 }
 
+func onModifyTableCharsetAndCollate(t *meta.Meta, job *model.Job) (ver int64, _ error) {
```

Comment:
If there is already data in the table, there will be no problem with the direct character set ?

Reply:
Can not understand what you say.

Reply:
Because TiDB treats all data charset as utf8mb4, so change to utf8 or utf8mb4 is safe.

Reply:
I understand.